adoc/extensions/sycl_khr_includes.adoc

```diff
@@ -0,0 +1,626 @@
+  [code]#sycl::errc#
+
+[[sec:khr-includes-half]]
+=== [code]#<sycl/khr/includes/half># header
+
+This header contains definition of [code]#half# class
+
+[[sec:khr-includes-vec]]
+=== [code]#<sycl/khr/includes/vec># header
+
+This header gives access to [code]#vec# class defined by <<sec:vector.type>>.
+It contains:
+
+- [code]#rounding_mode# enumeration
+- [code]#elem# struct and [code]#vec# class
+- Type aliases defined by the section 4.14.2.2 Aliases
+- [code]#__writeable_swizzle__# and [code]#__const_swizzle__# classes defined
+  by <<swizzled-vec-class>>
+
+Note that the simple swizzle functions ([code]#XYZW_SWIZZLE# and
+[code]#RGBA_SWIZZLE# defined by the table 123) are only available when the macro
+[code]#SYCL_SIMPLE_SWIZZLES# is defined before including
+[code]#<sycl/khr/includes/vec>#.
+
+[[sec:khr-includes-marray]]
+=== [code]#<sycl/khr/includes/marray># header
+
+This header gives access to math array types defined by <<sec:marray.type>>.
+It contains:
+
+- [code]#marray# class
+- Type aliases defined by the section 4.14.3.2 Aliases
+
+[[sec:khr-includes-atomic]]
+=== [code]#<sycl/khr/includes/atomic># header
+
+This header contains:
+- [code]#atomic_fence# function
+- [code]#memory_order#, [code]#memory_scope# enumerations
+- [code]#memory_order_traits# struct and [code]#atomic_ref# class as defined
+  by <<sec:atomic-references>>
+- [code]#atomic# class and free functions like [code]#atomic_store#,
+  [code]#atomic_fetch_and# as defined by the section 4.15.4 Atomic types
+  (deprecated)
+
+[[sec:khr-includes-stream]]
+=== [code]#<sycl/khr/includes/stream># header
+
+This header gives access to [code]#stream# class and related functionality
+described by <<subsec:stream>>. It contains:
+
+- [code]#stream_manipulator# enumeration and corresponding constants (like
+  [code]#flush# or [code]#endl#)
+- [code]#setprecision# and [code]#setw# functions
+- [code]#stream# class
+- [code]#template <typename T> const stream& operator<<(const stream& os, const T& rhs)#
+  operator
+
+[[sec:khr-includes-type-traits]]
+=== [code]#<sycl/khr/includes/type_traits># header
+
+This header contains:
+
+- [code]#is_property#, [code]#is_property_v#, [code]#is_property_of# and
+  [code]#is_property_of_v# defined by <<subsec:properties-interface>>
+- [code]#is_group# and [code]#is_group_v# defined by
+  <<subsec:group-type-traits>>
+- [code]#is_device_copyable# defined by <<subsec:is-device-copyable-type-trait>>
+- [code]#any_device_has#, [code]#any_device_has_v#, [code]#all_devices_have#
+  and [code]#all_devices_have_v# defined by <<sec:device-aspect-traits>>
+- [code]#remove_decoration# type trait
+
+[[sec:khr-includes-property-list]]
+=== [code]#<sycl/khr/includes/property_list># header
+
+This header contains:
+
+- [code]#property_list# class definition
+
+[[sec:khr-includes-functional]]
+=== [code]#<sycl/khr/includes/functional>#header
+
+This header contains:
+- [code]#plus#, [code]#multiplies#, [code]#bit_and#, [code]#bit_or#,
```

Comment:
Need a blank line before the bulleted list.  In addition, should each function be its own list item?  It's weird to have a bulleted list with only one item.

Reply:
The single-item list was split into multiple items in cfaf79c63c835204e2ce09b2c2a32d831d364738